lib/node/nodes/georeference-street-address.js

```diff
@@ -49,42 +53,55 @@ GeoreferenceStreetAddress.prototype.getGeocoderParams = function () {
     return geocoderParams;
 };
 
-var COLUMNS_TEMPLATE_REGEX = /\{\{([^\}]*)\}\}/g;
+var COLUMNS_TEMPLATE_REGEX = /\{\{([^\}]*)\}\}/;
 GeoreferenceStreetAddress.prototype.getStreetAddressColumnParam = function() {
     if (this.street_address_column) {
         return this.getFunctionParam('street_address');
     }
 
-    var columns = [];
-    var result = this.street_address_template.replace(COLUMNS_TEMPLATE_REGEX, function(s, column) {
-        columns.push(column.trim());
-        return '%s';
+    var columns_and_literals = this.street_address_template.trim().split(', ').map(function(fragment) {
+        fragment = fragment.trim();
+        if(COLUMNS_TEMPLATE_REGEX.test(fragment)) {
+            var column_match = COLUMNS_TEMPLATE_REGEX.exec(fragment);
+            return column_match[1].trim();
+        } else {
+            return '\'\'' + fragment + '\'\'';
```

Comment:
At the time we worked in this node, we couldn't use ES6 string interpolation/template literals. I encourage you to take advantage of them for better readability.